var/spack/repos/builtin/packages/hpctoolkit/package.py

```diff
@@ -139,9 +145,11 @@ class Hpctoolkit(AutotoolsPackage):
     depends_on("memkind", type=("build", "run"), when="@2021.05.01:")
     depends_on("papi", when="+papi")
     depends_on("libpfm4", when="~papi")
+    depends_on("mpi", when="+cray")
```

Comment:
Does it make sense instead:
```python
conflicts("~mpi", when="+cray", reason="activating +cray implies +mpi")
```
?

Reply:
Actually, our use of `+cray` is kinda unusual.  We use it as an
alternative to `+mpi` for some cases where the standard MPI doesn't
work (don't ask).  So, in this case, `+cray` depends on `mpi`
(cray-mpich), but it doesn't imply `+mpi`.